quickstarts/Code_Execution.ipynb

```diff
@@ -1498,6 +1529,22 @@
       "source": [
         "print(result.text)"
       ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "1a4c6e717f63"
+      },
+      "source": [
+        "## Next Steps\n",
+        "### Useful API references:\n",
+        "\n",
+        "Check the [Code execution documentation](https://ai.google.dev/gemini-api/docs/code-execution) for more details about the feature and in particular, the [recommendations](https://ai.google.dev/gemini-api/docs/code-execution?lang=python#code-execution-vs-function-calling) regarding when to use it instead of [function calling](https://ai.google.dev/gemini-api/docs/function-calling).\n",
+        "\n",
+        "### Continue your discovery of the Gemini API\n",
+        "\n",
+        "Learn how to control how the Gemini API can call your own functions using the [function calling](https://github.com/google-gemini/cookbook/blob/main/quickstarts/Function_calling.ipynb) feature, or discover how to control the model output in [JSON](https://github.com/google-gemini/cookbook/blob/main/quickstarts/JSON_mode.ipynb) or using an [Enum](https://github.com/google-gemini/cookbook/blob/main/quickstarts/Enum.*ipynb*)."
```

Comment:
Can you use relative links when referencing content in GitHub?

Doing so means that the links work:
* Across forks, branches, specific-commit views, etc
* In Colab
* In GitHub Codespaces

Reply:
I thought we wanted the opposite but sure, that a good reason.